features/step_definitions/applications/applications_steps.rb

```diff
@@ -168,7 +168,7 @@
 
 Given(/^I'm on that application page$/) do
   click_on 'Dashboard'
-  click_on 'Overview'
+  click_on 'default' # this is supposed to be the name of the service
```

Comment:
```suggestion
  click_on @application.service.name
```

Looks more dynamic 🤔 

No need to be today 👍